### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by eustas**
Like other -li projects, this one is minimalistic. Unlike other aperture images seen in logos here we have 7 leafs - no one manufactures such devices.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Foteli
 
+<p align="center"><img src="https://github.com/google/foteli/blob/main/doc/foteli.svg" alt="Foteli" width="64"></p>
```
Comment (sboukortt) [inline README.md]: Out of curiosity, any reason not to make this a relative path?

```suggestion
<p align="center"><img src="doc/foteli.svg" alt="Foteli" width="64"></p>
```

**Review by sboukortt — COMMENTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (eustas) [reply README.md]: Not sure, we always do so. I believe it is because `README` file can be embedded (not in iframe), e.g. on title page of repo.

Comment (sboukortt) [reply README.md]: I see, thanks. As far as I know, GitHub takes that into account when processing the Markdown. Let’s try with relative first and switch to absolute if there are any issues?

Comment (eustas) [reply README.md]: Sure, why not. Will update soon

Comment (eustas) [reply README.md]: Done. Even preview works

**Review by sboukortt — APPROVED** [review]

**Pull request merged**